custom_components/localtuya/common.py

```diff
@@ -219,6 +219,8 @@ async def async_added_to_hass(self):
         """Subscribe localtuya events."""
         await super().async_added_to_hass()
 
+        _LOGGER.debug("Adding %s with configuration: %s", self.entity_id, self._config)
```

Comment:
Do we really want to keep this debug message?

Reply:
I do like this debug print as it will tell us if a certain option is configured or not (it helps when a user thinks a he/she configured an option, but actually didn't).